test/e2e/e2e-profile/e2e-profile-test-runner/src/test/java/org/apache/skywalking/e2e/ProfileVerificationITCase.java

```diff
@@ -89,28 +86,6 @@ public void setUp() {
     public void verify() throws Exception {
         final LocalDateTime minutesAgo = LocalDateTime.now(ZoneOffset.UTC);
 
-        while (true) {
-            try {
-                final ResponseEntity<String> responseEntity = sendRequest(false);
-                LOGGER.info("responseEntity: {}", responseEntity);
-                assertThat(responseEntity.getStatusCode()).isEqualTo(HttpStatus.OK);
-                final List<Trace> traces = profileClient.traces(
-                        new TracesQuery()
-                                .start(minutesAgo)
-                                .end(LocalDateTime.now())
-                                .orderByDuration()
-                );
-                if (!traces.isEmpty()) {
-                    break;
-                }
-                Thread.sleep(10000L);
-            } catch (Exception ignored) {
-            }
-        }
-
-        // verify basic info
-        verifyServices(minutesAgo);
```

Comment:
You should still send traffic and verify service/instance/endpoint, just no metrics of `service/instance/endpoint`. If no traffic, then there is no trace to be profiled.